azure_jumpstart_ag/artifacts/PowerShell/AgConfig.psd1

```diff
@@ -18,6 +18,18 @@
         AgAppsRepo        = "C:\Ag\AppsRepo"
     }
 
+    # Required URLs
+    URL           = @{
+        chocoPackagesUrl = 'https://community.chocolatey.org/api/v2'
+        chocoInstallScriptUrl = 'https://chocolatey.org/install.ps1'
+        wslUbuntuUrl = 'https://aka.ms/wslubuntu'
+        wslStoreStorageUrl = 'https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi'
+        dockerUrl = 'https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe'
+        githubAPIUrl = 'https://api.github.com'
+        grafanaUrl = 'https://api.github.com/repos/grafana/grafana/releases/latest'
+        azurePortalUrl = 'https://portal.azure.com'
```

Comment:
Can we add the AKS EE installer link here too?

Reply:
![image](https://github.com/microsoft/azure_arc/assets/46730876/d39b3780-ea48-48b4-ae19-e908d0f0510b)